lib/slacked/slack_post.rb

```diff
@@ -1,9 +1,25 @@
 module Slacked
+  SLACK_PROFILE_IMAGE=':robot_face:'
+  SLACK_WEBHOOK_URL_KEY='SLACK_WEBHOOK'
+  SLACK_DEFAULT_MESSAGE_KEY='SLACK_DEFAULT_MESSAGE'
+  SLACK_CONFIG= {
+      icon_emoji: SLACK_PROFILE_IMAGE
+  }
+
   class << self
+    def post message = ENV[SLACK_DEFAULT_MESSAGE_KEY]
+      return false if message.nil? || message.empty?
+      notifier = slack_notifier
+      notifier.ping message, SLACK_CONFIG
+    end
+
+    def post_async message
+      Thread.start { post(message) }
```

Comment:
Do we need to close the database connection that is created by the Thread here? I've not used threads much but remember reading about each thread opening a DB connection that needs closing.